www/source/blog/2018-02-21-Habitat-Helm.html.md

```diff
@@ -22,28 +22,15 @@ And now, let's get started!
 * Habitat (`brew install habitat`) (check with `hab --version` to make sure you're on 0.54.0) Or, if you prefer the `curl | bash`, try `curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash`
 * Minikube [https://kubernetes.io/docs/tasks/tools/install-minikube/](https://kubernetes.io/docs/tasks/tools/install-minikube/) 
 * Kubectl (`brew install kubectl`) Alternate installation instructions available [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-* Go (`brew install golang`) Alternate installation instructions available [here](https://golang.org/doc/install#install) 
 * Helm (`brew install kubernetes-helm`) Additional information is available on their github repo at [https://github.com/kubernetes/helm](https://github.com/kubernetes/helm), and learn how to get started at [https://docs.helm.sh/using_helm/#quickstart-guide](https://docs.helm.sh/using_helm/#quickstart-guide)
 * Tiller [https://docs.helm.sh/using_helm/#initialize-helm-and-install-tiller](https://docs.helm.sh/using_helm/#initialize-helm-and-install-tiller) (i.e. just type `helm init`) 
-* Habitat Operator for Kubernetes: 
-
-```
-git clone https://github.com/kinvolk/habitat-operator.git
-cd habitat-operator
-make build
-make image
-```
-
-This will produce a `kinvolk/habitat-operator` image, which can then be deployed to your cluster.
-Read all about the Operator on Github at [https://github.com/kinvolk/habitat-operator/blob/master/README.md](https://github.com/kinvolk/habitat-operator/blob/master/README.md) 
 
 **Important Note** 
 
 Minikube uses its own Docker Engine, not your machine's Docker Engine. Habitat Studio uses your machine's Docker Engine. So, in order for your Minikube do be able to find the Helm package that you are about to build and export, you need to point your machine's Docker Engine to use Minikube's instead (once you've started your Minikube cluster). 
 
 So, for the purposes of this Helm demo to work locally, you now need to do: 
 
-* Be in the same directory as your compiled habitat-operator (i.e. habitat-operator)
 * Start minikube with RBAC enabled: `minikube start --extra-config=apiserver.Authorization.Mode=RBAC` 
```

Comment:
Considering the operator requires certain k8s version to work, I would pass in a specific k8s version. Either `1.8.4` or `1.9.0` for example.

Reply:
pass how? I'm not a minikube guy :)

Reply:
either:
`minikube start --kubernetes-version v1.8.0 --extra-config=apiserver.Authorization.Mode=RBAC`

or:
`minikube start --kubernetes-version v1.9.0`